core/common/src/main/java/alluxio/AlluxioEvent.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+/**
+ * Defines and Manages well-known Alluxio events.
+ */
+public enum AlluxioEvent {
+  // [1000,2000) Alluxio process events.
+  MasterProcessCreated(1000, "Alluxio master process created."),
+  JournalSystemStarted(1001, "Journal system started."),
```

Comment:
Will there be an example of when the message is different from the name of the enum? It seems like every event so far has a message which just mirrors the enum name.

Reply:
yeah seems redundant till now.